src/pages/Home/Home.jsx

```diff
@@ -1,10 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import EventCard from '../../components/EventCard/EventCard.jsx';
+import fetchLiked from '../../hooks/fetchLiked.js';
+import { APIS } from '../../config.js';
 import { S } from './Home';
 
 const Home = () => {
   const navigate = useNavigate();
+  const TOKEN = localStorage.getItem('accessToken');
+  const [checkLiked, setCheckLiked] = useState([]);
+  const [wishlist, setWishlist] = useState([]);
+  const [cardData, setCardData] = useState([]);
+  const setId = (data, event_id) => {
+    setCheckLiked(
+      fetchLiked(TOKEN, APIS.wishlist, data, event_id, setWishlist, getWishList)
+    );
+  };
+
+  const wishlistId = wishlist.map(({ event_id }) => event_id);
+
+  const getWishList = () => {
+    const url = `${APIS.wishlist}`;
+    if (TOKEN) {
+      fetch(url, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+          Authorization: TOKEN,
+        },
+      })
+        .then(response => response.json())
+        .then(result => {
+          setWishlist(result.wishlist);
+        });
+    } else return;
+  };
+
+  useEffect(() => {
+    fetch('data/homeEventCard.json')
+      .then(res => res.json())
+      .then(data => {
+        setCardData(data);
+      });
+  }, []);
+
+  useEffect(() => {
+    getWishList();
```

Comment:
```suggestion
    const url = `${APIS.wishlist}`;
    if (TOKEN) {
      fetch(url, {
        method: 'GET',
        headers: {
          'Content-Type': 'application/json;charset=utf-8',
          Authorization: TOKEN,
        },
      })
        .then(response => response.json())
        .then(result => {
          setWishlist(result.wishlist);
        });
```

getWishList 함수가 다른 곳에서 쓰이는 게 아니면 useeffect 안에서 사용해 주는 것도 좋겠네요